activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Add `with_all_variant_records` method to eager load all variant records on an attachment at once.
+    `with_attached_image` scope now eager loads variant records if using variant tracking.
+
+    *Alex Ghiculescu*
```

Comment:
It appears that these scopes aren't documented anywhere - they could really do with appearing in both the API and Guides documentation. Asking you to write the documentation for other PRs is a bit much so perhaps someone can create a documentation PR for these and other scopes like `with_all_rich_text`.

Reply:
Good call. I’ll make a follow up. Thanks for the review!